subjects/devops/array-selector/README.md

```diff
@@ -0,0 +1,95 @@
+## array-selector
+
+### Instructions
+
+Create a script `array-selector.sh`, which will have an array declared with the
```

Comment:
Explain somewhere that this array will work different than normal as it will start at position 1 instead of 0.

Reply:
I have added a sentence to make it more explicit - this can be extrapolated as well from the given examples